src/components/node-list/node-list-row.js

```diff
@@ -1,122 +1,145 @@
-import React from 'react';
+import React, { memo } from 'react';
 import { connect } from 'react-redux';
 import classnames from 'classnames';
+import changed from '../../utils/changed';
 import NodeIcon from '../icons/node-icon';
 import VisibleIcon from '../icons/visible';
 import InvisibleIcon from '../icons/invisible';
 import { getNodeActive } from '../../selectors/nodes';
 
-const NodeListRow = ({
-  container: Container = 'div',
-  active,
-  checked,
-  unset,
-  children,
-  disabled,
-  faded,
-  visible,
-  id,
-  label,
-  name,
-  kind,
-  onMouseEnter,
-  onMouseLeave,
-  onChange,
-  onClick,
-  selected,
-  type,
-  visibleIcon = VisibleIcon,
-  invisibleIcon = InvisibleIcon
-}) => {
-  const VisibilityIcon = checked ? visibleIcon : invisibleIcon;
+export const nodeListRowHeight = 36.59375;
```

Comment:
It might be worth getting this value from the DOM, or else add a comment in both the CSS and JS to ensure that updates to one get reflected in the other

Reply:
Yeah for sure, it just takes a little DOM gymnastics to get hold of the size. For now I've added some comments, but hopefully will add this later!